packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss

```diff
@@ -79,4 +79,16 @@
   &.is-docskit-visual-negative .docskit-visual--title::before {
     content: '🚫';
   }
+
+  &.is-docskit-visual-smiley-positive .docskit-visual--title::before {
+    content: '😄';
+  }
+
+  &.is-docskit-visual-smiley-neutral .docskit-visual--title::before {
+    content: '🙄';
```

Comment:
The :eyeroll: doesn't read at neutral to me; seems like the user is doing something wrong.

Reply:
@joejezowski-okta any recommendations here?